### ISSUE TITLE
`aria-live="polite"` should be an option to select.

### ISSUE THREAD

**Issue by asirihewage** [Pearson-Higher-Ed/alerts#24]
During accessibility audits, the recommended value for the aria-live attributes was 'assertive'. So, better if we can set `aria-live=""` as an option. Then developers can choose the Ally-priority of the alert.

```
<div role="alert">
      <TransitionGroup className="alertList" aria-live="polite" component="ul">
        {_renderAlert(alertList, handleClose)}
      </TransitionGroup>
 </div>
```

**FYI:** 
`aria-live="polite" `Any updates made to this region should only be announced if the user is not currently doing anything. live="polite" should be used in most situations involving live regions that present new information to users, such as updating news headlines. -

`aria-live="assertive"` Any updates made to this region are important enough to be announced to the user as soon as possible, but it is not necessary to immediately interrupt the user. live="assertive" must be used if there is information that a user must know about right away, for example, warning messages in a form that does validation on the fly.
